### PR TITLE
[NEW] DWS

### LINKED ISSUE #445: [NEW] Data Warehouse Service
https://confluence.tsi-dev.otc-service.com/display/DR/RM-4279-DWS-API
https://docs.otc.t-systems.com/en-us/api/dws/dws_02_0037.html

part of https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1805

### PR DESCRIPTION
**Pull request by Aloento**
fix #445 
--- PASS: TestDWS (2448.52s)
PASS

### CODE REVIEW

File: openstack/dws/v1/cluster/CreateCluster.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	// POST /v1.0/{project_id}/clusters
+	raw, err := client.Post(client.ServiceURL("clusters"), b, nil, &golangsdk.RequestOpts{
+		OkCodes: []int{200},
+	})
+	return ExtraClusterId(err, raw)
+}
+
+func ExtraClusterId(err error, raw *http.Response) (string, error) {
```
Comment (anton-sidelnikov) [inline openstack/dws/v1/cluster/CreateCluster.go]: Extract - misspell

File: openstack/dws/v1/snapshot/RestoreCluster.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		if err != nil {
+			return false, err
+		}
+
+		if current.Status == "AVAILABLE" && current.TaskStatus != "RESTORING" {
+			return true, nil
+		}
+
+		b, _ := json.MarshalIndent(current.ActionProgress, "", "  ")
+		print(string(b))
```
Comment (anton-sidelnikov) [inline openstack/dws/v1/snapshot/RestoreCluster.go]: remove this and others prints from waits

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: openstack/dws/v1/cluster/CreateCluster.go (hunk 1 above)
Comment (Aloento) [reply openstack/dws/v1/cluster/CreateCluster.go]: done

File: openstack/dws/v1/snapshot/RestoreCluster.go (hunk 2 above)
Comment (Aloento) [reply openstack/dws/v1/snapshot/RestoreCluster.go]: done

**Review by anton-sidelnikov — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**